src/IFeeCurrency.sol

```diff
@@ -30,41 +30,37 @@ import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
  * that omits deprecated fields and accommodates potential new recipients
  * that might become necessary on later blockchain implementations. Both
  * versions should be implemented to increase compatibility.
- * - Future Celo blockchain implementations might provide a way for plain
- * ERC-20 tokens to be used as gas currencies without implementing this
- * interface. If this alternative is preferable for you, please contact cLabs
- * before implementing this interface for your token.
  */
 interface IFeeCurrency is IERC20 {
-    /// @notice Called before transaction execution to reserve the maximum amount of gas
-    /// that can be used by the transaction.
-    /// - The implementation must deduct `value` from `from`'s balance.
-    /// - Must revert if `msg.sender` is not the zero address.
+    // @notice Called before transaction execution to reserve the maximum amount of gas
+    // that can be used by the transaction.
+    // - The implementation must deduct `value` from `from`'s balance.
+    // - Must revert if `msg.sender` is not the zero address.
```

Comment:
What is the harm in having this marked as a natspec comment? It will show up nicely in `forge docs` and probably other tools. Or should I just take this as a vote to do the natspec properly with plenty of `@` directives?